packages/core/apps/core-apps-server-internal/src/internal_types.ts

```diff
@@ -13,15 +13,15 @@ import type { UiSettingsRequestHandlerContext } from '@kbn/core-ui-settings-serv
  * Request handler context used by core's coreApp routes.
  * @internal
  */
-export interface InternalCoreAppRequestHandlerContext extends RequestHandlerContextBase {
+export interface InternalCoreAppsServiceRequestHandlerContext extends RequestHandlerContextBase {
```

Comment:
This was internal, so the change wasn't strictly necessary, but 👍 for consistency.